gen/definitions/interface_igmp.yaml

```diff
@@ -0,0 +1,46 @@
+---
+name: Interface IGMP
+path: Cisco-IOS-XE-native:native/interface/%s=%v/ip/Cisco-IOS-XE-igmp:igmp
+augment_path: Cisco-IOS-XE-native:native/interface/GigabitEthernet=%v/ip/Cisco-IOS-XE-igmp:igmp
+no_delete: true
+doc_category: Multicast
+attributes:
+  - yang_name: type
+    tf_name: type
+    description: Interface type
+    type: String
+    reference: true
+    enum_values:
```

Comment:
Let's limit it to the interface types we currently support for other such resources. Some of those types have a slightly different set of leafs, which we don't necessarily want to validate and test separately if not required.